python/strings.cpp

```diff
@@ -32,4 +41,26 @@ std::string proc_allocation_string(const arb::proc_allocation& a) {
     return s.str();
 }
 
+std::string schedule_explicit_string(const explicit_schedule_shim& e) {
```

Comment:
Define these inside `event_generator.cpp` to avoid having to have `event_generator.hpp`

Reply:
Done.